models/stg_tiktok_ads.sql

```diff
@@ -0,0 +1,37 @@
+{{ config(enabled=var('ad_reporting__tiktok_ads_enabled')) }}
+
+with base as (
+
+    select *
+    from {{ ref('tiktok_ads__ad_adapter')}}
+
+), fields as (
+
+    select
+        company_name as account_name,
+        cast(advertiser_id as {{ dbt_utils.type_string() }}) as account_id,
+        cast(date_day as date) as date_day,
+        base_url,
+        url_host,
+        url_path,
+        utm_source,
+        utm_medium,
+        utm_campaign,
+        utm_content,
+        utm_term,
+        cast(campaign_id as {{ dbt_utils.type_string() }}) as campaign_id,
+        campaign_name,
+        cast(ad_group_id as {{ dbt_utils.type_string() }}) as ad_group_id,
+        ad_group_name as ad_group_name,
+        'TikTok Ads' as platform,
+        sum(coalesce(clicks, 0)) as clicks,
+        sum(coalesce(impressions, 0)) as impressions,
+        sum(coalesce(spend, 0)) as spends
```

Comment:
should be "as spend"

Reply:
@sziep great catch! 👀 

We can push this quick fix through. However, since you are a great eagle eyed viewer I would welcome you to insert a suggestion if you would like to be a contributor on this repo!

To do so, you can simple select the `insert a suggestion` icon in Github and propose the change. 
![image](https://user-images.githubusercontent.com/74217849/146800466-afe49df1-a17a-4cdb-8409-23739dee9975.png)

Otherwise, we will make this change and extremely appreciate you looking through this PR! 🎉

Reply:
Just applied the change. Thanks so much @sziep